Makefile

```diff
@@ -222,7 +222,7 @@ aws-run-migrator:
 	@cat migration-results | grep "success"
 
 aws-run-clear-dev-data: 
-	@rm clear-db-results || true
+	@touch clear-db-results || true
```

Comment:
These are very different, what's the reasoning?

Reply:
We need a file to output the results to. I kept seeing a message that the rm <filename> couldn't be found. (We see it in the migrations as well). I guess I could delete the file afterwards as well?

Reply:
Okay updated to also remove the file afterwards. This doesn't change the functionality but it removes an un-needed warning message about the file not existing.